man/systemd-analyze.xml

```diff
@@ -1091,7 +1091,7 @@ io.systemd.credential:vmm.notify_socket=vsock-stream:2:254570042
     </refsect2>
 
     <refsect2>
-      <title><command>systemd-analyze chid</command></title>
+      <title><command>systemd-analyze chid <optional>/sys/class/drm/&lt;card&gt;/edid</optional></command></title>
```

Comment:
We don't use `<>` to mark up params. Please use `<replaceable>card</replaceable>` or `<replaceable>CARD</replaceable>` instead.